webui dir is made already by "quarkus create app" (#170)
quarkiverse/quarkus-quinoa: docs/modules/ROOT/pages/index.adoc

[#getting-started]
== Getting Started

> `quarkus create app ...`, above, will have created a Web UI directory in `src/main/webui`. This directory will contain your NodeJS Web application code with a `package.json`. The location is configurable, the directory could be outside the Quarkus project as soon as the file are available at build time.

Reviewer: ```suggestion
If not yet created by the tooling, you will need  a Web UI directory in `src/main/webui`. This directory will contain your NodeJS Web application code with a `package.json`. The location is configurable, the directory could be outside the Quarkus project as soon as the file are available at build time.
```
